core/camel-core-reifier/src/main/java/org/apache/camel/reifier/ResumableReifier.java

```diff
@@ -47,6 +49,17 @@ protected ResumeStrategy resolveResumeStrategy() {
             String ref = parseString(definition.getResumeStrategy());
             strategy = mandatoryLookup(ref, ResumeStrategy.class);
         }
+
         return strategy;
     }
+
+    protected LoggingLevel resolveLoggingLevel() {
+        LoggingLevel loggingLevel = parse(LoggingLevel.class, definition.getLoggingLevel());
+
+        if (loggingLevel == null) {
+            loggingLevel = LoggingLevel.WARN;
```

Comment:
Here default value is WARN but in the model it is ERROR.

Reply:
Ahh, good catch. I think we can default to ERROR.